content/blog/elm-review-v2/index.md

```diff
@@ -0,0 +1,269 @@
+---
+title: elm-review v2!
+date: '2020-04-30T00:00:00.000Z'
+---
+
+Today I am very excited to release `elm-review` 2.0.0 and to share its new features!
+
+tl;dr: Here is the list of the introduced features:
+
+#### New review capabilities!
+
+- Project rules!
+- Reporting errors for `elm.json`
+- Reading the dependencies' `docs.json` file
+- Visiting the README.md file
+- Visiting the comments and documentation
+- Adding helpers
+
+#### Easier to use!
+
+- Much faster, and with a watch mode
+- Configuring exceptions
+- A `fix-all` flag
+- Better default folder structure
+- Tests included by default
+- More rules to start with
+
+## Sorry but... what is elm-review?
+
+If you missed the [initial announcement](/announcing-elm-review/) or if your memory is unclear, let me explain.
+
+`elm-review` is a static code analysis tool for Elm, which looks at your Elm project, and reports patterns that you configured it to find using "rules" in a friendly Elm compiler-like way.
+
+It is highly customizable because you can write your own rules. Since these are written in Elm, you don't need to know a different language to do so, and you can even publish them as Elm packages!
+
+I put a lot of effort into making a [**very nice API**](https://package.elm-lang.org/packages/jfmengels/elm-review/2.0.0/Review-Rule) for you to use ([here](https://github.com/jfmengels/review-simplification/blob/master/src/NoBooleanCaseOf.elm) [are](https://github.com/jfmengels/review-debug/blob/master/src/NoDebug/Log.elm) [some](https://package.elm-lang.org/packages/jfmengels/elm-review/2.0.0/Review-Rule#withSimpleExpressionVisitor) [examples](https://package.elm-lang.org/packages/jfmengels/elm-review/2.0.0/Review-Rule#withSimpleImportVisitor)), even going through the lengths of discovering new Elm techniques (expect to see blog posts on the **phantom builder pattern** on this blog), and I am very happy with the result!
+
+To some, `elm-review` looks like a linter. Which isn't necessarily wrong, since you can enable rules that help improve the quality of your code. If you dive a bit further, you will find out that it can enforce coding conventions for your team, and create new guarantees that the Elm compiler can not give you.
```

Comment:
This entire paragraph describes exactly what a linter is in my mind, though `elm-review` is probably more geared towards customization than the average linter 🙂 I would probably just leave this out, since it's already covered pretty well by the preceding paragraphs 😊

Reply:
Yeah, I guess I phrased it wrong :thinking: 

I've changed it to the following:
```
To some, `elm-review` looks like a linter. Which isn't necessarily wrong, since you can enable rules that help improve the quality of your code and enforce coding conventions for your team.

If you dive a bit further, you will find out that it can create new guarantees that the Elm compiler can not give you.

For instance you can write rules that
...
```

If it feels like a linter to you because that's what ESLint does too, it's because ESLint is also more than a linter (and should probably have a different name :man_shrugging:).

Reply:
I mean, the description for the original lint goes ([from wiki](https://en.wikipedia.org/wiki/Lint_(software))): "lint, or a linter, is a tool that analyzes source code to flag programming errors, bugs, stylistic errors, and suspicious constructs.[1] The term originates from a Unix utility that examined C language source code.[2]"

But any linter still has to be configured with rules, so to my mind, `elm-review` is still a linter, even if it is a very customizable one. I'll let you decide on what to say though, but I do still think this paragraph just repeats what you've just written 🙂

Reply:
Though I guess it depends on what you consider to be in the "errors, bugs, stylistic errors" category 😛

Reply:
Yeah, the definition is very loose.

> But any linter still has to be configured with rules

Have you met https://standardjs.com/ or https://github.com/psf/black? :smile: